apps/communications/dialer/test/marionette/lib/views/contact_list/accessors.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
Yes, [that file](https://github.com/mozilla-b2g/gaia/blob/master/apps/communications/contacts/test/marionette/lib/contacts.js) to remove the ambiguity